style.css

```diff
@@ -0,0 +1,703 @@
+@import url('https://fonts.googleapis.com/css2?family=Arimo:ital,wght@1,700&family=Poppins:wght@300;600;800&display=swap');
```

Comment:
Please note that your font-weights are not being applied correctly because you have not imported all of them. You are only importing 300;600 and 800, I would highly suggest to import all the font-weights for the Poppins family so that your website can resemble the Figma design 🤗